repo.go

```diff
@@ -49,7 +49,15 @@ type LocalStore interface {
 	// If paths is empty, all staged target files will be walked.
 	WalkStagedTargets(paths []string, targetsFn TargetsWalkFunc) error
 
+	// IsStaged determines if a metadata file is currently staged, to avoid incrementing
+	// version numbers repeatedly while staged.
+	IsStaged(string) bool
```

Comment:
Can we name arguments in the interface?

Additionally, one thing I'd like to gradually clean up is naming of role names vs. manifest/file names. Could we name the arguments or the function name to reflect that this takes in a manifest/file name? e.g. `FileIsStaged(name string)` or `IsStaged(fileName string)` or whatever you think makes the most sense. It looks like this function could take in staged targets files as well as staged manifests.

Reply:
done